sorc/build_ufs.sh

```diff
@@ -32,10 +32,38 @@ COMPILE_NR=0
 CLEAN_BEFORE=YES
 CLEAN_AFTER=NO
 
-./tests/compile.sh "${MACHINE_ID}" "${MAKE_OPT}" "${COMPILE_NR}" "intel" "${CLEAN_BEFORE}" "${CLEAN_AFTER}"
-mv "./tests/fv3_${COMPILE_NR}.exe" ./tests/ufs_model.x
-mv "./tests/modules.fv3_${COMPILE_NR}.lua" ./tests/modules.ufs_model.lua
-cp "./modulefiles/ufs_common.lua" ./tests/ufs_common.lua
-cp "./modulefiles/ufs_common_spack.lua" ./tests/ufs_common_spack.lua
+if [[ "${MACHINE_ID}" != "noaacloud" ]]; then
+  ./tests/compile.sh "${MACHINE_ID}" "${MAKE_OPT}" "${COMPILE_NR}" "${CLEAN_BEFORE}" "${CLEAN_AFTER}"
+  mv "./tests/fv3_${COMPILE_NR}.exe" ./tests/ufs_model.x
+  mv "./tests/modules.fv3_${COMPILE_NR}.lua" ./tests/modules.ufs_model.lua
+  cp "./modulefiles/ufs_common.lua" ./tests/ufs_common.lua
+fi
+
+if [[ "${MACHINE_ID}" == "noaacloud" ]]; then
+  
+  if [[ "${PW_CSP}" == "aws" ]]; then
+    # TODO: This will need to be addressed further when the EPIC stacks are available/supported.
+    module use /contrib/spack-stack/envs/ufswm/install/modulefiles/Core
+    module load stack-intel
+    module load stack-intel-oneapi-mpi
+    module load ufs-weather-model-env/1.0.0
+    # TODO: It is still uncertain why this is the only module that is 
+    # missing; check the spack build as this needed to be added manually.
+    module load w3emc/2.9.2 # TODO: This has similar issues for the EPIC stack.
+    module list
+  fi
+
+  cd "${cwd}/ufs_model.fd" # TODO: This is redundant; see line #28.
+  export CMAKE_FLAGS="${MAKE_OPT}"
+  ./build.sh 
+  mv "${cwd}/ufs_model.fd/build/ufs_model" "${cwd}/ufs_model.fd/tests/ufs_model.x"
+ 
+  # TODO: This is hack? Where is this step performed in the build system?
+  mkdir -p "${HOMEgfs}/exec"
+  cp "${cwd}/ufs_model.fd/tests/ufs_model.x" "${HOMEgfs}/exec/ufs_model.x" 
```

Comment:
```suggestion
```
Indeed a hack and not necessary.
This is performed in ......

Reply:
I still don't see (cannot find) where this step is performed.